tests/admin_views/admin.py

```diff
@@ -594,7 +594,7 @@ def get_search_results(self, request, queryset, search_term):
         try:
             search_term_as_int = int(search_term)
             queryset |= self.model.objects.filter(age=search_term_as_int)
```

Comment:
Maybe this line should be moved out of the try block? I guess `int()` is the only thing expected to raise `ValueError`.